app/views/instructor_alert_mailer/email.html.haml

```diff
@@ -0,0 +1,12 @@
+%link{rel: 'stylesheet', href:'/mailer.css'}
+%p.paragraph
+	Hello,
+	%p.paragraph
+		The following article being worked on as a part of your course has received a “Did You Know” nomination:
+		%a.link{href: @alert.article.url}= @alert.article.title
+
+	%p.paragraph
+		We recommend checking the submission and encouraging your student to monitor the nomination and respond to feedback.
+		
+	%p.paragraph
+		Thank you and please let us know if you need any support.
```

Comment:
Just a couple things to add to this. Please include the link to the Did You Know sheet that was in the issue.
https://github.com/WikiEducationFoundation/WikiEduDashboard/issues/2635#issuecomment-477388352

Additionally, we'll want to include some kind of signature. You can take a look at the other mailers to see how we do signatures. Let me know if you can't find it.

Reply:
@bwreid how do i get link for ` Did You Know `, i am not sure about it,